app/consumer-democracy/app.go

```diff
@@ -427,7 +427,7 @@ func New(
 		authtypes.FeeCollectorName,
 	)
 
-	// Setting the staking keeper is only needed for standalone to consumer changeover chains
+	// Setting the standalone staking keeper is only needed for standalone to consumer changeover chains
```

Comment:
Nomenclature is strange to me, why not just refer to it as Sovereign to Consumer? Or is there a precedent that was decided already for this?

Reply:
Yea there is a precedent here: https://github.com/cosmos/interchain-security/pull/757#discussion_r1142231475. "Sovereign" could make it seem like consumers have no sovereignty